images/Dockerfile.rhel7

```diff
@@ -1,4 +1,9 @@
-FROM fedora:29
+# Current LOWEST version is 2.4.57.
+# Version could be patched by OS.
+ARG VERSION=2.4.57
+ARG OS=fedora:34
```

Comment:
I wonder why the Dockerfile's name is (and was) .rhel7 if it's got fedora in it

Reply:
I have no clue, I just copied the behavior as closely as possible from master.

Compare: https://github.com/openshift/openldap/blob/master/2.4.41/Dockerfile.rhel7#L1

Reply:
I am just brave and copy paste that one here: 

```Dockerfile
FROM registry.access.redhat.com/ubi8/ubi
```